FindYourFriends/screens/CreateUserScreen.jsx

```diff
@@ -21,7 +19,7 @@ const CreateUser = (props) => {
     email: "",
     password: "",
   });
-  const [addUser, { data }] = useMutation(facade.ADD_USER);
```

Comment:
gonna cause a conflict. 
Like this on ErrorHandling branch atm: 
`const [addUser, { loading, error, data, called }] = useMutation(
    facade.ADD_USER
  );`

Reply:
Yeah that's why this branch should be merged first. Because you know what's needed and what isn't from the hook

Reply:
Then you can also apply the new error handling to the 2 new checks I made